shardingsphere-features/shardingsphere-master-slave/shardingsphere-master-slave-common/src/test/java/org/apache/shardingsphere/masterslave/yaml/swapper/MasterSlaveRuleConfigurationYamlSwapperTest.java

```diff
@@ -89,4 +102,27 @@ private void assertMasterSlaveRuleConfiguration(final MasterSlaveRuleConfigurati
         assertThat(group.getMasterDataSourceName(), is("master_ds"));
         assertThat(group.getSlaveDataSourceNames(), is(Arrays.asList("slave_ds_0", "slave_ds_1")));
     }
+    
+    @Test
+    public void assertGetTypeClass() {
+        MasterSlaveRuleConfigurationYamlSwapper masterSlaveRuleConfigurationYamlSwapper = getMasterSlaveRuleConfigurationYamlSwapper();
+        Class<MasterSlaveRuleConfiguration> actual = masterSlaveRuleConfigurationYamlSwapper.getTypeClass();
+        assertTrue(actual.isAssignableFrom(MasterSlaveRuleConfiguration.class));
+    }
+    
+    @Test
+    public void assertGetOrder() {
+        MasterSlaveRuleConfigurationYamlSwapper masterSlaveRuleConfigurationYamlSwapper = getMasterSlaveRuleConfigurationYamlSwapper();
+        int actual = masterSlaveRuleConfigurationYamlSwapper.getOrder();
+        assertEquals(MasterSlaveOrder.ORDER, actual);
```

Comment:
Please use assertThat instead of assertEquals

Reply:
already edited